CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 ## 1.18.0 
 ## October 9, 2024
 
+SECURITY:
+
+* secrets/identity: A privileged Vault operator with write permissions to the root namespace's identity endpoint could escalate their privileges to Vault's root policy (CVE-2024-9180) [HCSEC-2024-21](https://discuss.hashicorp.com/t/hcsec-2024-21-vault-operators-in-root-namespace-may-elevate-their-privileges/70565)
```

Comment:
Should "to the root namespace's identity endpoint" instead use the specific endpoint(s)? I'm also fine with this as-is though.